04/WeatherAPI.js

```diff
@@ -0,0 +1,24 @@
+class WeatherAPI {
+	constructor() {
+		this.KEY_API = '6265f4d03b4b474f88d58705e51823bc';
+	}
+
+	_fetch(latitude, longitude) {
```

Comment:
Podkreślenie sugeruje, że to metoda prywatna (https://kursjs.pl/kurs/obiekty/obiekty-enkapsulacja#private-public).
W Twoim przypadku odpytujesz ja na zewnątrz wiec nie powinna być prywatna.